lib/action_list_view.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:bender/bender_vm.dart';
+import 'package:bendroid/main.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class ActionListView extends StatefulWidget {
+  final String prName;
+
+  final Map prHistory;
+  final String title;
+  ActionListView(
+      {Key key, this.title = 'Bendroid Actions', this.prName, this.prHistory})
```

Comment:
instead of prName pass prModel(final url and final title, int for useTime) and instead of prHistory list of all prModels